README.rst

```diff
@@ -446,8 +442,8 @@ GeometryFilter
     from rest_framework_gis.filterset import GeoFilterSet
 
     class RegionFilter(GeoFilterSet):
-        slug = filters.CharFilter(name='slug', lookup_type='istartswith')
```

Comment:
Are you sure removing `lookup_type` doesn't break compatibility with older django-filter versions?